docs/reference/mapping/types/semantic-text.asciidoc

```diff
@@ -82,10 +82,8 @@ PUT my-index-000003
 
 `inference_id`::
 (Required, string)
-{infer-cap} endpoint that will be used to generate the embeddings for the field.
-This parameter cannot be updated.
-Use the <<put-inference-api>> to create the endpoint.
-If `search_inference_id` is specified, the {infer} endpoint defined by `inference_id` will only be used at index time.
+{infer-cap} endpoint that will be used to generate embeddings for the field. By default, `.elser-2-elasticsearch` is used. This parameter cannot be updated. Use the <<put-inference-api>> to create the endpoint.
+If `search_inference_id` is specified, the {infer} endpoint will only be used at index time.
```

Comment:
I'd recommend to put each sentence to a new line, so the diff will be easier to read when we update the text.
```suggestion
{infer-cap} endpoint that will be used to generate embeddings for the field.
By default, `.elser-2-elasticsearch` is used.
This parameter cannot be updated.
Use the <<put-inference-api>> to create the endpoint.
If `search_inference_id` is specified, the {infer} endpoint will only be used at index time.
```